activities/communication/run-a-community-call.md

```diff
@@ -0,0 +1,109 @@
+
+2. Substance of the call
+
+## 1. Opening the call
+
+Call-in link: [https://meet.google.com/tfd-acmn-tkb[(https://meet.google.com/tfd-acmn-tkb)
+
+Agenda link: [https://hackmd.io/-OegeqvoThCbAsw3c3gIjw?edit](https://hackmd.io/-OegeqvoThCbAsw3c3gIjw?edit)
+
+### No-show
+
+If nobody joins at the call starting time, wait at least 15 minutes before closing the call. If possible, leave the call open the entire hour for potential late drop-ins.
+
+### Introduction round
+
+The chair:
+
+* welcomes everyone to the community call on behalf of the Foundation for Public Code
+* briefly mentions what is on the agenda for today (without being so specific that discussions start)
+* notes that the call will be minuted, but not recorded, and the minutes published on our blog. Everyone on the call will get the chance to review and fix the minutes before they're published
+* asks everyone to introduce themselves - who are you, do you represent an organization, what's your interest in public code or the topic for this call?
+* starts with themselves
+* asks someone else to go next (for example, someone they know to be excited about the topic, or a friend of the Foundation, or an extrovert)
+* keeps track of who's introduced themselves and nudges anyone who hasn't by the end
+
+### Short update from the Foundation for Public Code
+
+The chair:
+
+* thanks everyone for introducing themselves
+* asks if everyone's familiar with the Foundation
+* gives optional Foundation intro
+* gives update since the last community call (from the notes document)
+
+> #### Optional intro to the Foundation
+
+> The Foundation for Public Code is a member-driven, member-funded and member-governed association of public organizations.
+
+> We work to enable open and collaborative public-purpose software for public organizations internationally.
+
+> The Foundation for Public Code supports software at the codebase level through codebase stewardship and by publishing the Standard for Public Code.
+
+## 2. Substance of the call
+
+The chair explains:
+
+* the substantive agenda
+* any timekeeping plans (to manage expectations - is this a free form or strict agenda?)
+
+The chair then either:
+
+* hands over to the (first) substantive expert
+* facilitates discussion of the substantive agenda themselves
+
+## 3. Closing the call
+
+When either the time is up, or all topics have been exhausted, the chair:
+
+* asks if anyone has anything else to add or ask before closing the call
+* reminds everyone that there is another call in two weeks and asks for suggested topics for that call
+* explains that the minutes will be sent out for review
+* thanks everyone for participating in the call
+
+## After the call
+
+The notetaker reworks the call notes into a call summary.
+
+The notetaker shares the call summary with people who were on the call for amendment and approval before:
+
+* publishing on [blog.publiccode.net](https://blog.publiccode.net/)
```

Comment:
FYI, this blog is not up yet.

This bears the question, should this already be up there if it is unoperationalisable.